workflows/cloudify_system_workflows/snapshots/utils.py

```diff
@@ -34,20 +34,25 @@
 SCHEMA_SCRIPT = '/opt/manager/resources/cloudify/migrations/schema.py'
 
 
-class DictToAttributes(object):
-    def __init__(self, dic):
-        self._dict = dic
-
-    def __getattr__(self, name):
-        return self._dict[name]
+class DictToAttributes(dict):
```

Comment:
Since you are not doing a deep copy anymore, I think this is not needed?

Reply:
The class DictToAttributes, as I understand it, is created in order to allow the action `config.<key>` instead of `config[key]`. 
What do you mean by not needed?